store/postgres/alerts.go

```diff
@@ -24,16 +25,16 @@ func (r Repository) Migrate() error {
 	return nil
 }
 
-func (r Repository) Create(alert *model.Alert) (*model.Alert, error) {
+func (r Repository) Create(alert *domain.Alert) (*domain.Alert, error) {
```

Comment:
let's update the values of referenced param instead of returning another `*domain.Alert` like we've refactored in other pkgs as well?